lib/miq_automation_engine/service_models/miq_ae_service_authentication_rhsm.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceAuthenticationRhsm < MiqAeServiceAuthentication
```

Comment:
They would if someone wrote their own automate code and just called `.authentications` directly.